frontend/public/components/dashboard/dashboards-page/cluster-dashboard/health-item.tsx

```diff
@@ -226,6 +227,49 @@ export const PrometheusHealthItem = withDashboardResources<PrometheusHealthItemP
   },
 );
 
+export const CRDHealthItem = withDashboardResources<CRDHealthItemProps>(
+  ({ watchK8sResource, stopWatchK8sResource, resources, subsystem, models }) => {
```

Comment:
I'd prefer to use [useK8sWatchResource](https://github.com/openshift/console/blob/master/frontend/public/components/utils/k8s-watch-hook.ts) hook instead of `watchK8sResource`/`stopWatchK8sResource` which I want to remove gradually. `useK8sWatchResource` is Firehose reimplementation into hook form

Reply:
Can we consider using [useK8sWatchResources](https://github.com/openshift/console/blob/master/frontend/public/components/utils/k8s-watch-hook.ts#L104) ?

We can completely omit the `additionalResources` props and pass all resources at once.